README.md

```diff
@@ -6,86 +6,107 @@
 ![formatting](https://github.com/google/temporian/actions/workflows/formatting.yaml/badge.svg)
 ![publish](https://github.com/google/temporian/actions/workflows/publish.yaml/badge.svg)
 
-Temporian is an open-source Python library for preprocessing ⚡ and feature engineering 🛠 temporal data 📈 for machine learning applications 🤖. It is a library tailor-made to address the unique characteristics and complexities of time-related data, such as time-series and transactional data.
+**Temporian** is an library for **safe**, **simple** and **effcient** preprocessing and feature engineering of temporal data in machine learning pipelines in Python. Temporian supports multivariate time-series, multivariate time-sequences, event logs, and cross-source event streams.
 
-> Temporal data is any form of data that represents a state in time. In
-> Temporian, temporal datasets contain [events](https://temporian.readthedocs.io/en/stable/user_guide/#events-and-eventsets), which consists of
-> values for one or more attributes at a given timestamp. Common
-> examples of temporal data are transaction logs, sensor signals, and
-> weather patterns. For more, see
-> [What is Temporal data](https://temporian.readthedocs.io/en/stable/user_guide/#what-is-temporal-data).
+Temporian is to [temporal data](https://temporian.readthedocs.io/en/stable/user_guide/#what-is-temporal-data) what Pandas is to tabular data.
 
 ## Key features
 
-- **Unified data processing** 📈: Temporian operates natively on many forms
-  of temporal data, including multivariate time-series, multi-index
-  time-series, and non-uniformly sampled data.
+- **Support most type of temporal data** 📈: Handles both uniformly sampled and
+non-uniformly sampled data, both single-variate and multivariate data, both flat
+ and multi-index data, and both mono-source and multi-source non-synchronized
+ events.
 
+- **Protect from unwanted future leakage** 😰: Unless explicitly specified with
+`tp.leak`, features computation cannot depend on future data, thereby preventing
+ unwanted, hard-to-debug, and potentially costly future leakage.
+
+ - **Optimized for Temporal data** 🔥: Temporian's core computation is
+ implemented in C++ and optimized for temporal data. Temporian can be more than
+ 1,000x faster than non-temporal-specific libraries when operating on temporal
+ data.
+
+ - **ML library agnostic:** Temporian does not perform any machine learning
```

Comment:
How about

**Integrates with all ML libraries:** 
Temporian does not perform any ML model training - instead it integrates seamlessly with any ML library, such as PyTorch, Scikit-Learn, Jax, or TensorFlow.

Reply:
Or `Easy to integrate into an existing ML ecosystem` ?

Reply:
I think we should definitely mention XGBoost here :)

Reply:
*Apply all proposed comments.
*Adding XGBoost and YDF :)